travel_platform/Html/main.html

```diff
@@ -0,0 +1,111 @@
+{% extends 'blog/base.html' %}
+{% block content %}
+<header>
+  <div class="row head-bkg">
+    <div class="col-md-offset-3 col-md-6 head-text text-center">YOUR<br>TRAVEL</div>
+  </div>
+</header>
+
+<div class="row">
+  <div class="col-md-offset-4 col-md-8"><h2 class="text-center white">Common information</h2>
+  </div>
+
+  <div class="col-md-4 choose">
+    <a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/planet.jpg" alt="planet">
+      <span class="text-center">Choose a city</span>
+    </a>
+  </div>
+
+  <div class="col-md-8">
+    <div class="col-md-4 choose" id="museums"><a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/museum.png" alt="museum">
+        <span class="text-center">Museums</span></a>
+    </div>
+
+    <div class="col-md-4 choose" id="attractions"><a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/attraction.jpg" alt="attraction">
+        <span class="text-center">Attractions</span></a>
+    </div>
+
+    <div class="col-md-4 choose" id="entertainment"><a href="https://lviv.itea.ua/">
+      <img class="img-responsive" src="pics/entertainment.jpg" alt="entertainment">
+        <span class="text-center">Entertainment</span></a>
+    </div>
+
+    <div class="col-md-4 choose" id="hotels"><a href="https://lviv.itea.ua/">
```

Comment:
last three col-md-4 divs are inside second col-md-8 div. So they take 4 from 12 columns for each of them, but that 12 columns are inside col-md-8 div. That's why these columns are smaller than normal  columns of whole page.